doc/adding_lints.md

```diff
@@ -86,6 +86,8 @@ test. That allows us to check if the output is turning into what we want.
 
 Once we are satisfied with the output, we need to run
 `tests/ui/update-all-references.sh` to update the `.stderr` file for our lint.
+Please note that, we should run `TESTNAME=ui/foo_functions cargo uitest`
+before running `tests/ui/update-all-references.sh`.
```

Comment:
maybe add "every time", as this is the frustrating part.
You change something, but the `.stderr` doesn't change until you run `uitest` _again_

Reply:
Thanks! I added "every time".